package.json

```diff
@@ -29,6 +29,7 @@
     "@nestjs/swagger": "^7.4.0",
     "@nestjs/throttler": "^6.0.0",
     "@prisma/client": "^5.17.0",
+    "aws-sdk": "^2.1663.0",
```

Comment:
please use "@aws-sdk/client-s3" instead. 
bcoz, we are not going to use most of the AWS services.

Reply:
@bhupend-deshmukh ^^